tests/unit/onyxTest.js

```diff
@@ -24,11 +24,37 @@ Onyx.init({
 describe('Onyx', () => {
     let connectionID;
 
+    /** @type OnyxCache */
+    let cache;
+
+    beforeEach(() => {
+        cache = require('../../lib/OnyxCache').default;
+    });
+
     afterEach(() => {
         Onyx.disconnect(connectionID);
         return Onyx.clear();
     });
 
+    it('should remove key value from OnyxCache/Storage when set is called with null value',
+        () => Onyx.set(ONYX_KEYS.OTHER_TEST, 42)
```

Comment:
```suggestion
    it('should remove key value from OnyxCache/Storage when set is called with null value', () => Onyx.set(ONYX_KEYS.OTHER_TEST, 42)
```
Just to match the style elsewhere.

Reply:
Doen with latest commit